data/partials/mainnav.yaml

```diff
@@ -305,8 +302,12 @@ main:
     url: "tracing/"
     pre: "nav_tracing"
     children:
+      # Enable tracing
+      - name: "Send traces to Datadog"
+        url: "tracing/send_traces/"
+
       # Languages
-      - name: "Setup"
+      - name: "Enable traces"
```

Comment:
should we put this section before the `Send traces to Datadog`?

Reply:
not yet. I think this might be an iterative update though, where we combine both of these and make it all language/container based anyway. this is something we're working with design on.